fdbserver/Status.actor.cpp

```diff
@@ -499,6 +499,11 @@ struct RolesInfo {
 				    maxTLogVersion - version - SERVER_KNOBS->STORAGE_LOGGING_DELAY * SERVER_KNOBS->VERSIONS_PER_SECOND);
 			}
 
+			TraceEventFields const& batchLatencyMetrics = metrics.at("BatchLatencyMetrics");
```

Comment:
This is the SS role, which won't have these metrics. You can remove this section here.